### PR TITLE
feat(Docchi): regexp update and visual refresh

### PR DESCRIPTION
**Pull request by Meff1u**
## Description 
Added some URLs to iframe regexp and and moved details on presence to make it look better

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
- 
With privacy turned on
-

### CODE REVIEW

**Comment by Meff1u** [comment]
i have no clue why there are commits from february

File: websites/D/Docchi/presence.ts [hunk 1]
```diff
@@ -139,6 +140,7 @@ presence.on("UpdateData", async () => {
 							},
 						];
 					}
+					presenceData.state = "Docchi.pl";
```
Comment (darkvillager2) [inline websites/D/Docchi/presence.ts:143]: Not really sure if this is advertising, if it is it isn't allowed.

**Review by darkvillager2 — CHANGES_REQUESTED** [review]
Other than this LGTM

File: websites/D/Docchi/presence.ts (hunk 1 above)
Comment (Meff1u) [reply websites/D/Docchi/presence.ts:143]: It’s not meant to look like an advertisement. I set it up due to changing the presence name to 'watching series,' which means there's no longer 'Watching Docchi' while watching, and nothing besides this 'Docchi.pl' state indicates the site you're watching on. However, if this is not allowed, I’ll remove it :)

**Review by Bas950 — APPROVED** [review]

File: websites/D/Docchi/presence.ts (hunk 1 above)
Comment (Bas950) [reply websites/D/Docchi/presence.ts:143]: It's klnda meh to me...

Comment (Meff1u) [reply websites/D/Docchi/presence.ts:143]: fine, then im gonna remove it

**Review by theusaf — APPROVED** [review]

**Pull request merged**